documentation/book/cluster-operator.adoc

```diff
@@ -537,7 +537,7 @@ The first option supports default values.
 [[resources_json_config]]
 ===== Resource limits and requests
 
-It is possible to configure {ProductPlatformName} resource limits and requests on for the `kafka`, `zookeeper` and `topicOperator` objects in the `Kafka` resource and for for the `spec` object of the `KafkaConnect resource.
+It is possible to configure {ProductPlatformName} resource limits and requests on for the `kafka`, `zookeeper` and `topicOperator` objects in the `Kafka` resource and for for the `spec` object of the `KafkaConnect` resource.
```

Comment:
please delete "for" (requests on for the `kafka`) and extra "for" in the next line (and for for the `spec`)

Reply:
Missing comma after `zookeepter`